jsx/Form.js

```diff
@@ -1187,14 +1235,33 @@ class NumericElement extends Component {
     let disabled = this.props.disabled ? 'disabled' : null;
     let required = this.props.required ? 'required' : null;
     let requiredHTML = null;
+    let errorMessage = null;
+    let elementClass = 'row form-group';
+
+    // Add required asterix
+    if (required) {
+      requiredHTML = <span className="text-danger">*</span>;
+    }
+
+    // Add error message
+    if (this.props.errorMessage) {
+      errorMessage = <span>{this.props.errorMessage}</span>;
+      elementClass = 'row form-group has-error';
+    }
+
+    const label = this.props.label && (
+      <label className="col-sm-3 control-label" htmlFor={this.props.id}>
+        {this.props.label}
+        {requiredHTML}
+      </label>
+    );
+
+    const columnSize = this.props.label ? 'col-sm-9' : 'col-sm-12';
 
     return (
-      <div className="row form-group">
-        <label className="col-sm-3 control-label" htmlFor={this.props.id}>
-          {this.props.label}
-          {requiredHTML}
-        </label>
-        <div className="col-sm-9">
+      <div className={elementClass}>
+        {label}
+        <div className={columnSize}>
```

Comment:
not in **24.1**

_added to improve form functionality and UI_

to include as **CBIG override** with plans to push to **main** in some capacity